hexa/files/api.py

```diff
@@ -57,6 +59,7 @@ def create_bucket(bucket_name):
             }
         ]
         bucket.patch()
+        _init_bucket_data(bucket_name)
```

Comment:
I would not do that here since this api is made to be generic and not related to workspaces